### ISSUE TITLE
Bug: Build output not respecting file extensions in import statements

### ISSUE THREAD

**Issue by sachaw** [fullstack-build/tslog#250]
**Describe the bug**
This library clearly adheres to the ESM requirement for having file extensions present in import/export statements, however that appears to be striped in the build output.

Take the file `tslog/dist/esm/index.js` for example, the generated output is as follows:
```ts
import { BaseLogger } from "./BaseLogger";
import BrowserRuntime from "./runtime/browser/";
import NodeRuntime from "./runtime/nodejs/";
export * from "./interfaces";
export * from "./BaseLogger";
export { BrowserRuntime, NodeRuntime };
...
```

but should be:
```ts
import { BaseLogger } from "./BaseLogger.js";
import BrowserRuntime from "./runtime/browser/index.js";
import NodeRuntime from "./runtime/nodejs/index.js";
export * from "./interfaces.js";
export * from "./BaseLogger.js";
export { BrowserRuntime, NodeRuntime };
...
```

**Comment by terehov** [comment]
Thanks! Your fast, that slipped in the yesterday's release. Will fix it now.

**Comment by terehov** [comment]
Fix: `v4.8.5`
